crates/common/consensus/src/deneb/beacon_state.rs

```diff
@@ -99,20 +118,25 @@ pub struct BeaconState {
     // Eth1
     pub eth1_data: Eth1Data,
     pub eth1_data_votes: VariableList<Eth1Data, U2048>,
+    #[serde(with = "serde_utils::quoted_u64")]
     pub eth1_deposit_index: u64,
 
     // Registry
     pub validators: VariableList<Validator, U1099511627776>,
+    #[serde(with = "quoted_u64_var_list")]
     pub balances: VariableList<u64, U1099511627776>,
 
     // Randomness
     pub randao_mixes: FixedVector<B256, U65536>,
 
     // Slashings
+    #[serde(with = "quoted_u64_fixed_vec")]
     pub slashings: FixedVector<u64, U8192>,
 
     // Participation
+    #[serde(deserialize_with = "deserialize_string_vec_to_variable_list")]
     pub previous_epoch_participation: VariableList<u8, U1099511627776>,
+    #[serde(deserialize_with = "deserialize_string_vec_to_variable_list")]
```

Comment:
```suggestion
    #[serde(with ="quoted_u64_var_list")]
    pub previous_epoch_participation: VariableList<u8, U1099511627776>,
    #[serde(with ="quoted_u64_var_list")]
```


can't we just use quoted_u64_var_list? instead of rolling our whole?

Reply:
Will not compile in this case as the iterator must yeild `u8` not `u64`

Reply:
could you call your deserializer `quoted_u8_var_list`, then to be consistent with the other deserializer's we use